front/docker/Dockerfile.playwright

```diff
@@ -0,0 +1,19 @@
+FROM alpine AS exec_as_builder
+
+RUN apk add build-base
+COPY front/docker/exec-as.c .
+RUN gcc -std=c99 -static -o /exec-as exec-as.c
+
+FROM mcr.microsoft.com/playwright
+
+COPY --from=exec_as_builder /exec-as /
+WORKDIR /app
+COPY front front
+COPY tests tests
+WORKDIR /app/front
+RUN yarn install
+RUN yarn playwright install --with-deps
```

Comment:
This should probably not be dependent on the *whole* front, only its dependencies

Reply:
Done, thanks